pkg/noderesourcetopology/filter.go

```diff
@@ -156,3 +146,34 @@ func resourceFoundOnNode(resName v1.ResourceName, wantQuantity resource.Quantity
 	}
 	return false
 }
+
+func getPodResource(pod *v1.Pod) v1.ResourceList {
```

Comment:
Better to move the function to `https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/util/resource.go`. Then it can be used by other plugins if needed.